cloud/openstack/os_network.py

```diff
@@ -125,6 +137,9 @@ def main():
         admin_state_up=dict(default=True, type='bool'),
         external=dict(default=False, type='bool'),
         state=dict(default='present', choices=['absent', 'present']),
+        provider_network_type=dict(required=False),
+        provider_physical_network=dict(required=False),
```

Comment:
We should make this a list of choices since they are known ahead of time. e.g., choices=['flat', 'vlan', 'vxlan', 'gre']